README.md

```diff
@@ -26,6 +26,26 @@ At that point, you can then use PyScript components in your html page. PyScript
 
 Check out the [pyscriptjs/examples](pyscriptjs/examples) folder for more examples on how to use it, all you need to do is open them in Chrome.
 
+If you are trying to open the examples in the Chrome then try to change the `pyscript.js` and `pyscript.css` local reference to the CDN one that mentioned above (https://pyscript.net/alpha/pyscript.js, https://pyscript.net/alpha/pyscript.css).
```

Comment:
I think the place for an explanation on how to run pyscript trusting on the web components doesn't fit on the Readme anymore. We're working on adding docs to Pyscript and this kind of information belongs there. If you want to help maybe take a look in the PR is progress: https://github.com/pyscript/pyscript/pull/262

Reply:
+1 . We need to sit down and think about what's the best way to walk the users through that experience.